test/kubetest/utils.go

```diff
@@ -254,6 +263,18 @@ func defaultNSCEnv() map[string]string {
 	}
 }
 
+func noNseHeal(k8s *K8s) *pods.NSMgrPodConfig {
```

Comment:
Why is it called `noNseHeal`? I mean what `nse` stands for?

Reply:
This method returns value of variable test/integration/recover_death_handling_test.go:20(see removed line). I've used the same name of the variable